bin/direxec

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+
+mode = os.path.basename(__file__)
+
+dir_odoo = '/opt/odoo'
+dir_common = os.path.join(dir_odoo, 'common', '%s.d' % mode)
+dir_custom = os.path.join(dir_odoo, 'custom', '%s.d' % mode)
+
+files = [
+    os.path.join(dir_common, d) for d in os.listdir(dir_common)
+]
+
+try:
+    files += [
+        os.path.join(dir_custom, d) for d in os.listdir(dir_custom)
+    ]
+except OSError:
+    pass
+
+for command in sorted(files):
+    if os.access(command, os.X_OK):
+        exit_code = os.system(command)
+        if exit_code:
+            raise EnvironmentError(
```

Comment:
To clarify - the point of this instead of subprocess was to hopefully run under the same process ID. Given the below information, I will use `os.execvp` here instead of the recommended subprocess. 

The intent is to run all of the commands under the same pid right? I'm honestly thinking I'm still missing a nuance of why, so I'm just going with the flow. Is this so that Docker properly issues the exit command to the commands, which would allow for a proper shutdown instead of a hard one?